content/en/docs/authentication/service_accounts.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Service accounts
+linkTitle: Service accounts
+description: Use the Axway CLI service-account command to create and manage service accounts.
+weight: 10
+date: 2021-08-30
+siblings_only: true
+---
+
+The {{% variables/axway_cli_prod_name %}} `service-account` command allows you to create and manage service accounts, generate public/private keypairs, and assign teams.
+
+You must authenticate into a platform account to use the `service-account` command. If your platform account is not the default account, you need to pass in the `--account` argument or set your platform account as the default for your session using the `axway auth switch` command.
+
+## Usage
+
+```
+axway service-account
+
+axway service-account <command> [options]
+```
+
+## Commands
+
+* `add-team` - Add a team to a service account
+* `create` - Create a service account
+* `generate-keypair` - Create a pem formatted public/private key pair
+* `ls`, `list` - List all service accounts
+* `rm`, `remove` - Remove a service account
+* `remove-team` - Remove a team from a service account
+* `roles` - View available service account roles
+* `update` - Update a service account
+* `v`, `view` - View service account details
+
+### add-team
+
+Adds an existing team to a service account. When assigning a team, you must also specify the team role.
+
+```
+axway service-account add-team <client-id/name> <team-guid/name> <role>
+```
+
+You may specify the service account by client id or name as well as the team by guid or name. The available team roles can be discovered by running `axway service-account roles`.
```

Comment:
I missed one... change to 
Run `axway service-account roles` to discover the available team roles.